src/tram_stop_mapper/gtfs_package.py

```diff
@@ -83,9 +111,13 @@ def trip_data_and_stops_by_trip_id(self):
 
         trip_data_by_trip_id: defaultdict[str, dict[str, Any]] = defaultdict(dict)
         for trip_id, trip_row in self.trips.iterrows():
+            service_id = trip_row["service_id"]
             for name, value in trip_row.items():
-                trip_data_by_trip_id[trip_id][name] = value
-
+                if name != "service_id":
+                    trip_data_by_trip_id[trip_id][name] = value
+            trip_data_by_trip_id[trip_id]["service_days"] = list(
+                service_id_to_days.get(service_id)
+            )
```

Comment:
Maybe something like this?
```suggestion
                if name == "service_id":
                    trip_data_by_trip_id[trip_id]["service_days"] = list(
                        service_id_to_days.get(service_id)
                    )
                else:
                    trip_data_by_trip_id[trip_id][name] = value
```